### ISSUE TITLE
[Enhancement] Toolbar and header visual changes

### ISSUE THREAD

**Issue by jakobe** [kirbydesign/designsystem#3040]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
UX has made some small visual changes to the toolbar on mobile + desktop (before and after scroll):

- [x] Toolbar on mobile should always be xxl/56px (effectively reverts changes done in #2219)
- [x] Toolbar on desktop should always be xxxxxl/72px (new size should be added to sizes-ramp)
- [x] Toolbar separator bottom-line should always be visible on desktop
   - [x] Except when [tab-navigation](https://cookbook.kirby.design/#/examples/page/tab-navigation) is used 
- [x] Toolbar background should be Kirby black + 1% transparent
   - [ ]  Please note: the design specs shows a translucent toolbar, but as this has implications when using tab navigation, this change is postponed to #3247
- [x] ~~Toolbar buttons should have 16px spacing on the right on mobile and 24px on desktop~~
       _**Please note:** This enhancement will be implemented in a separate issue when we fix the spacing of toolbar action buttons and space available for toolbar title (is currently partly overlapped by toolbar buttons)_ 

 

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
